.rubocop.yml

```diff
@@ -1,260 +1,17 @@
-# This (http://c2.com/cgi/wiki?AbcMetric) is super obnoxious
-AbcSize:
-  Enabled: false
-
-AccessorMethodName:
-  Enabled: false
-
-Alias:
-  Enabled: false
+require:
+  - rubocop/rspec/focused
 
 AllCops:
-  RunRailsCops: true
-
-AmbiguousOperator:
-  Enabled: false
-
-AmbiguousRegexpLiteral:
-  Enabled: false
-
-ArrayJoin:
-  Enabled: false
-
-AsciiComments:
-  Enabled: false
-
-AsciiIdentifiers:
-  Enabled: false
-
-AssignmentInCondition:
-  Enabled: true
-
-Attr:
-  Enabled: false
-
-BlockNesting:
-  Enabled: false
-
-BracesAroundHashParameters:
-  Enabled: false
-
-CaseEquality:
-  Enabled: false
-
-CharacterLiteral:
-  Enabled: false
-
-ClassLength:
-  Enabled: false
-
-ClassVars:
-  Enabled: false
-
-CollectionMethods:
-  PreferredMethods:
-    find: detect
-    reduce: inject
-    collect: map
-    find_all: select
-
-ColonMethodCall:
-  Enabled: false
-
-CommentAnnotation:
-  Enabled: false
-
-ConditionPosition:
-  Enabled: false
-
-CyclomaticComplexity:
-  Enabled: false
-
-Delegate:
-  Enabled: false
-
-DeprecatedClassMethods:
-  Enabled: false
+  TargetRubyVersion: 2.3
```

Comment:
I wasn't sure. We need something akin to this in here because without this, RuboCop tells us that the `VERSION` constant should be frozen, but if I do that locally then it complains about the unnecessary version freezing because of the `# frozen_string_literal: true` comment above it.

I'm not sure what about my local setup was differing from that of Travis CI (locally, I didn't need this line for `rubocop` to pass).

Since we're testing on Ruby 2.2 and AR 5 supports Ruby 2.2 I thought it made sense to keep this as low as possible, but I don't have strong feelings either way.